### DISCUSSION TITLE
react-resizeable  意外触发了排序

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#42581] [Q&A]
有人遇到没

**Comment by BoyYangzai** [comment]
> 有人遇到没

可以提供一个复现demo吗

**Comment by afc163** [comment]
建议先搜一下：https://github.com/search?q=repo%3Aant-design%2Fant-design+resizeable+%E6%8E%92%E5%BA%8F&type=issues

**Reply by vaynevayne** [reply]
找到了, 不过搜  react-resizeable  啥也搜不到
